src/sentry/search/utils.py

```diff
@@ -316,6 +318,23 @@ def parse_team_value(projects: Sequence[Project], value: Sequence[str], user: Us
     ).first() or Team(id=0)
 
 
+def get_teams_for_users(projects: Sequence[Project], users: Sequence[User]) -> list[Team]:
+    user_ids = (
+        [users.id]
+        if not (users is None or isinstance(users, list))
+        else [u.id for u in users if u is not None]
```

Comment:
Hmm, would it make sense to remove the `isinstance(users, list)` check here and just ensure the callers of this method passes in a list?

Its generally better to avoid `isinstance` checks and trust the caller is properly passing in the correct arguments.